apps/Parser/TextParser.v3

```diff
@@ -43,7 +43,41 @@ class LexerChoiceCombo(lexers: Array<Lexer>) extends Lexer {
 
 // A graph-based deterministic finite automata lexer.
 class DfaLexer extends Lexer {
-	// TODO
+	// DFA implementation would require a state transition table
```

Comment:
Given that `TextParser` is really just a demo application, nothing depends on it, so I don't think this change is a priority. I think I'd prefer this PR without this change.